fprime-vxworks/VxWorks/Log.cpp

```diff
@@ -25,6 +30,15 @@ namespace Os {
         POINTER_CAST a8,
         POINTER_CAST a9
     ) {
-        ::logMsg(const_cast<char*>(fmt), a1, a2, a3, a4, a5, a6);
+
+        // copy the possibly transient string to a circular buffer
+        // this relies on the VxWorks log task to keep up with
+        // the circular buffer
+
+        strncpy(logBuffers[logEntry],fmt,FW_LOG_TEXT_BUFFER_SIZE);
```

Comment:
Can you please void out the return by changing `strncpy` to `(void)strncpy` ?

Reply:
Done!